tfhe-zk-pok/src/proofs/pke.rs

```diff
@@ -822,13 +838,17 @@ pub fn verify<G: Curve>(
 
         if e(c_h - G::G1::GENERATOR.mul_scalar(p_h), G::G2::GENERATOR)
             + e(G::G1::GENERATOR, c_hat_t - G::G2::GENERATOR.mul_scalar(p_t)).mul_scalar(w)
-            == e(pi_kzg, g_hat_list[1] - G::G2::GENERATOR.mul_scalar(z))
+            == e(
+                pi_kzg,
+                G::G2::projective(g_hat_list[1]) - G::G2::GENERATOR.mul_scalar(z),
+            )
         {
             Ok(())
         } else {
             Err(())
         }
     } else {
+        // PERF: rewrite as multi_mul_scalar?
```

Comment:
should it be done ?

Reply:
yeah, but it's not urgent since it's for the slow verify, which isn't used at the moment afaik

Reply:
ok I'll create an issue for it then

Reply:
https://github.com/zama-ai/tfhe-rs-internal/issues/616